subprojects/core/src/main/java/org/gradle/api/internal/tasks/SnapshotTaskInputsBuildOperationType.java

```diff
@@ -161,6 +161,8 @@
              */
             String getPropertyNormalizationStrategyName();
 
+            Set<String> getPropertyAttributes();
```

Comment:
Missing `@since` Javadoc, and documentation in general.

We should also clarify what the semantics of `getPropertyNormalizationStrategyName` are for 7.3 and up.